plugins/common/common.cpp

```diff
@@ -109,6 +120,9 @@ class AuiToolBar : public wxAuiToolBar
 	void OnTool( wxCommandEvent& event );
 
 	DECLARE_EVENT_TABLE()
+
+private:
+	std::unordered_map<int, void*> m_aObjects;
```

Comment:
This element is used only locally and seems to store only `wxObject*` elements, i would prefer to use this as data type so the casts can be avoided